### ISSUE TITLE
Versions 1.0.0+ are missing TypeScript support

### ISSUE THREAD

**Issue by vladanpaunovic** [honeybadger-io/honeybadger-js#121]
## What are the steps to reproduce this issue?
1. Run any typescript project
2. Install honeybadger-js v1.0.0+

## What happens?
1. The file `honeybadger-js.d.ts` is missing, therefore, there are no types for `Honeybadger` instance.

## What were you expecting to happen?
Have types as they were until the version bump.

## Any logs, error output, etc?
```
error TS7006: Parameter 'notice' implicitly has an 'any' type.
types/lib.d.ts(58,31): error TS7016: Could not find a declaration file for module 'honeybadger-js'. '/finderly/Shpock-Web-3/node_modules/honeybadger-js/dist/honeybadger.cjs.js' implicitly has an 'any' type.
  Try `npm install @types/honeybadger-js` if it exists or add a new declaration (.d.ts) file containing `declare module 'honeybadger-js';`
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @shpock/web@3.0.0 check: `tsc --noEmit`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @shpock/web@3.0.0 check script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by joshuap** [comment]
Thanks, looking into this now.

**Comment by joshuap** [comment]
@vladanpaunovic I released v1.0.2 with the fix in #122. Hopefully that does it. If not, re-open this issue and we'll try again. :)

**Comment by vladanpaunovic** [comment]
@joshuap just bumped the version to 1.0.2 and I can confirm that everything works properly.
Thank you very much for fast response and fixing this! 🚀
